index.html

```diff
@@ -11,211 +11,251 @@
-                <i class="see-project"></i>See this project &#8594;</a>
+<div class="box">
+  <a href="#" class="button2">
+    <i class="getcv"></i>Get my CV &#8595;</a>
+</div>
+
+<section class="languages">
+  <div class="lang">Languages</div>
+  <ul>
+    <li class="lang1">&#x2022;Javascript (ES5/ES6)</li>
+    <li class="lang2">&#x2022;HTML</li>
+    <li class="lang3">&#x2022;CSS</li>
+  </ul>
+  <div class="figure1">
+    <img src="images/figure1.png">
+  </div>
+  </div>
+</section>
+
+<section class="frameworks">
+  <div class="framework-list">Frameworks</div>
+  <ul>
+    <li class="fram1">&#x2022;Bootstrap</li>
+    <li class="fram2">&#x2022;Ruby on rails</li>
+    <li class="fram3">&#x2022;RSpec</li>
+    <li class="fram4">&#x2022;Capibara</li>
+    <li class="fram5">&#x2022;Selenium</li>
+  </ul>
+  <div class="figure2">
+    <img src="images/ic_framework.png">
+  </div>
+  </div>
+</section>
+
+<section class="Skills">
+  <div class="skill1">Skills</div>
+  <ul>
+    <li class="skill2">&#x2022;Codekit</li>
+    <li class="skill3">&#x2022;Codepen</li>
+    <li class="skill4">&#x2022;Github</li>
+    <li class="skill5">&#x2022;GitLab</li>
+    <li class="skill6">&#x2022;Terminal</li>
+  </ul>
+  <div class="figure3">
+    <img src="images/ic_skills.png">
+  </div>
+</section>
+
+<section aria-label="Contact-me">
+    <div class="container-contact-me">
+      <h2 class="get-started">Get started</h2>
+      <p class="second-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
+        <div class="input-label">
+          <label class="label">Your name</label>
+          <input class="input" name="name" id="name" type="name" pattern="[A-Za-z/ ]{0,30}" required> 
         </div>
-    </section>
-    
-    <section class="aboutme">
-        <h2>About</h2>
-    </section>
-
-    <section class="about2">
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod.</p>
-    </section>
-
-    <div class="box">
-        <a href="#" class="button2">
-            <i class="getcv"></i>Get my CV   &#8595;</a>
-    </div>
-      
-    <section class="languages">
-        <div class="lang">Languages</div>
-                <ul>
-                    <li class="lang1">&#x2022;Javascript (ES5/ES6)</li>
-                    <li class="lang2">&#x2022;HTML</li>    
-                    <li class="lang3">&#x2022;CSS</li>
-                </ul>
-                <div class="figure1">
-                    <img src="images/figure1.png">
-                </div>
+        <div class="input-label">
+          <label class="label">Your Email</label>
+        <input class="input" name="Email" id="email" type="email" required>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), all inputs should have the correct placeholder text. You should add placeholder="Your email" to the input instead of relying on the label.

Reply:
Thank you @laiifuu  for your review and suggestions! I have a question regarding the placeholders in Name and email: we decided to put label so the "your name" and "your email" text could be visible while writing like it is in the figma design. So to put a placeholder we should delete these labels and put placeholder as we did in "Enter your message here..."? It will look different from the figma design but we also understand those are project requirements... so it would be great to clarify that! Thank you.

Reply:
@Maedea9 Good morning, could you text me on slack ?